systemtest/cmd/apmbench/main.go

```diff
@@ -59,10 +60,55 @@ func BenchmarkOTLPTraces(b *testing.B) {
 	})
 }
 
+func BenchmarkAgentGo(b *testing.B) {
+	benchmarkAgent(b, `traces/go.*.ndjson`)
```

Comment:
The prefix `traces` is configurable in the Intake Receiver - how is this kept in sync with this functionality always reading from `traces/` ?

Reply:
The two folders needn't be named the same, however when we copy the generated traces over to `apmbench`, we'll need to match the path that is specified in the benchmarks. We should strive to keep the names in sync since I don't think we gain anything by naming things differently. What do you think?

Reply:
I've changed this slightly to have the tests specify a shortened path and not have to specify the full path.